### PR TITLE
Fix occ config commands

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @Xenopathic @DeepDiver1975 

Found these issues while writing the documentation for the commands:
https://github.com/owncloud/documentation/pull/1332

### CODE REVIEW

**Comment by karlitschek** [comment]
looks good :+1:

File: core/command/base.php [hunk 1]
```diff
@@ -27,6 +27,8 @@
 use Symfony\Component\Console\Output\OutputInterface;
 
 class Base extends Command {
+	protected $defaultOutputFormat = 'plain';
```
Comment (Xenopathic) [inline core/command/base.php]: This should be a `const`, with any references to it as `static::DEFAULT_OUTPUT_FORMAT` so subclasses can override the constant.

**Comment by nickvergessen** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
Tested and still works :*1:

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Pull request merged**